input/pagecontent/reference.md

```diff
@@ -2027,13 +2027,6 @@ The units of measure can be set by assigning a coded value to a Quantity:
 
 <pre><code>* &lt;Quantity&gt; = <span class="optional">{CodeSystem}|{version string}</span>#{code} <span class="optional">"{units display string}"</span></code></pre>
 
-A Quantity can also be bound to a value set:
```

Comment:
Any reason you cut this out?  I really like taking a jab at FHIR for the way they defined Quantity.

Reply:
It doesn't feel like it really fits in the `Assignments with the Quantity Data Type` section, since it isn't about assignment rules, and we mention that `Quantities` can be bound to in the `Binding Rules` section, so it felt unnecessary. But we do lose a good jab.